### ISSUE TITLE
Annotation completeness

### ISSUE THREAD

**Issue by kermitt2** [howisonlab/softcite-dataset#512]
Considering PMC1747177, we have 9 mentions of `GenomeRNAi` in the complete article, but there are 7 mentions with quote in the .ttl file. 

Regarding the two 2 other mentions not recorded, if I am not wrong, one is in the title of the article and one in the title of a bibliographical references. In general it seems that mentions in the bibliographical references are not annotated (for instance `E-RNAi` in the bib ref. 13 is not annotated as a software_name). Should it be also the case for the title and other header metadata? 

In general, what is the policy regarding annotation completeness? 

For a ML approach, it's better normally to have a larger context that the one provided by the quotes (typically the whole paragraph) and we need "negative" examples in a proportion which is as realistic as possible with the actual distribution of software mention names (otherwise we will tend to find software everywhere). 

What are the constraints/reliability regarding the places where there is no software mention - in particular a complete paragraph and a complete article?

**Comment by jameshowison** [comment]
Yes, we don't code for the title or references independently, just the main body of text.

I hope that by locating the quotes in the converted PDF we can then expand or contract the context as needed.

For negative examples I had been thinking we'd select a random sample of sentences/nits

**Comment by kermitt2** [comment]
no problem for ignoring title and references, GROBID is very reliable for finding explicitly this. However, looking a bit more, it's still not very clear what is considered and not, even in the body. 

For PMC5226643, I saw only 6 mentions of `pMELTS` out of 8 are coded in the .ttl file. The two mentions not coded are in a section title and in a figure caption. 

I could consider to keep systematically:
* only abstract in the article header zone (are keywords also ignored?)
* all the body, without figures, but including reference callout 

to ignore systematically:
* all the header except the abstract section
* all running headnotes, page, acknowledgement and bibliographical reference

and to ignore only if there is no mention/quotes in those sections:
* figure and tables
* section header
* footnotes

Does it sound like a safe strategy?

**Comment by jameshowison** [comment]
Basically yes, if the students have coded things in those locations (and keywords should be ignored) then that strategy would ignore them, right?  Sounds good.  Hmmm, that means that we need to be careful comparing manual queries of the ttls and the expected counts in the test set (which would have some parts of the paper ignored).  I think that's ok.

Why would one ignore footnotes if no mention? Wouldn't one also want negative examples from them?
